scripts/augment_patroni_configuration.py

```diff
@@ -22,13 +22,14 @@
     command: '/usr/bin/pgbackrest --stanza=poddb --delta restore --log-level-stderr=info'
     keep_data: True
     no_params: True
+    no_master: 1
```

Comment:
I know 1 is a synonym for True but the previous two flags use True so perhaps we should here

Reply:
I agree with the ugliness, I'm following the documentation here:

https://patroni.readthedocs.io/en/latest/replica_bootstrap.html#building-replicas

I might throw in a PR upstream